tests/epics/signal/test_signals.py

```diff
@@ -769,6 +769,15 @@ async def test_str_enum_returns_enum(ioc: IOC):
     assert val == "Bbb"
 
 
+async def test_str_datatype_in_CA_signals_mbbo_mbbi(ioc: IOC):
```

Comment:
This test seems to be testing for both ioc.protocols, so the name could be adjusted